techmatters/hrm
# CHI-1749: Import progress get endpoint

## Description

* Add GET endpoint to retrieve the import progress for an account
* Remove the 'total' property from the KHP import state - it wasn't useful
* Add a 'remaining' property to the KHP import state, when thisa hits zero, a new batch can be started
* Fixed a typo in the `resource-import-consumer` status - not sure if it would affect anything but it bothers me

### Checklist


### Related Issues
CHI-1749

### Verification steps

* Automated tests verify this. Will be verified E2E when we add the lambda that calls tjhe KHP import API
